scripts/lib/ExtensionNameValidator.js

```diff
@@ -0,0 +1,31 @@
+/** @param {string} extensionName */
+exports.isValidExtensionName = (extensionName) => {
+  if (extensionName.length === 0) return false;
+
+  {
```

Comment:
No need for this block?

Reply:
It is not needed but I like to make blocks like that to get rid of the temporary variable as soon as I stop needing it.